+++
title = "DMM"
author = ["John Doe"]
draft = false
+++

## DMM <code>[0/97]</code> {#dmm}


### Information {#information}


#### Journal Articles {#journal-articles}

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Engheta, N. Circuits with Light at Nanoscales: Optical Nanocircuits Inspired by Metamaterials.

    Science 317, 1698–1702 (2007).

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Pendry, J. B., Schurig, D. & Smith, D. R. Controlling Electromagnetic Fields. Science 312, 1780–1782 (2006).

    Invisibility paper

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Roberts, G., Ballew, C., Zheng, T., Camayd‐Muñoz, P. & Faraon, A. Inverse design approaches for volumetric meta‐optics. in Plasmonics: Design, Materials, Fabrication, Characterization, and Applications XVIII vol. 11462 114621D (International Society for Optics and Photonics, 2020).

    Contains the original cube in light fn out image

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Tikhomirov, G. et al. DNA‐based programming of quantum dot valency, self‐assembly and luminescence. Nat. Nanotechnol. 6, 485–490 (2011).

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Cademartiri, L. & Bishop, K. J. M. Programmable self‐assembly. Nat. Mater. 14, 2–9 (2015).

    This explains why we need to embed more complex recognition abilities into nanoparticles

<!--list-separator-->

-  NEXT Rizzuto, F. J., Trinh, T. & Sleiman, H. F. Molecular Printing with DNA Nanotechnology. Chem 6, 1560–1574 (2020).

    This paper shows how DNA can guide nanoparticle assembly

<!--list-separator-->

-  DNA software tools in esteemed journals

    1.  Benson, E. et al. DNA rendering of polyhedral meshes at the nanoscale. Nature 523, 441–444 (2015).
    2.  Veneziano, R. et al. Designer nanoscale DNA assemblies programmed from the top down. Science 352, 1534–1534 (2016).
    3.  Han, D. et al. DNA Origami with Complex Curvatures in Three‐Dimensional Space. Science 332, 342– 346 (2011).
    4.  Huang, C.‐M., Kucinic, A., Johnson, J. A., Su, H.‐J. & Castro, C. E. Integrating computer‐aided engineering and computer‐aided design for DNA assemblies. bioRxiv 2020.05.28.119701 (2020) <10.1101/2020.05.28.119701>.

<!--list-separator-->

-  Bass, M. et al. Handbook of Optics, Third Edition Volume II: Design, Fabrication and Testing,Sources and Detectors, Radiometry and Photometry. (McGraw‐Hill Education, 2009).

    Optic materal properties book

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Synthesis of silica nanocubes by sol–gel method. Mater. Lett. 59, 4013–4015 (2005).

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Yang, X. et al. Anatase TiO2 nanocubes for fast and durable sodium ion battery anodes. J. Mater. Chem. A 3, 8800–8807 (2015).

    How we can get Ti02 nanocubes for the DMM project

<!--list-separator-->

-  Johnson, P. B. & Christy, R. W. Optical Constants of the Noble Metals. Phys. Rev. B 6, 4370–4379 (1972).

    Silvers Properties

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Della Giovampaola, C. & Engheta, N. Digital metamaterials. Nat. Mater. 13, 1115–1121 (2014).     :ATTACH:

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Why silver is a useful material for giving interesting optical properties (if it is because complex n what does this allow?)

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Reread DNA Origami Paper

    Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-29 Thu 11:26]</span></span>

<!--list-separator-->

- <span class="org-todo todo HOLD">HOLD</span>  Burns, J. R., Lamarre, B., Pyne, A. L. B., Noble, J. E. & Ryadnov, M. G. DNA Origami Inside‐Out Viruses. ACS Synth. Biol. 7, 767–773 (2018)

    Single scaffold near-perfect DNA origami cube.

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Petersen, P., Tikhomirov, G. & Qian, L. Information‐based autonomous reconfiguration in systems of interacting DNA nanostructures. Nat. Commun. 9, 5362 (2018).

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Zhitomirsky, D. et al. Colloidal Quantum Dot Photovoltaics: The Effect of Polydispersity. Nano Lett. 12, 1007–1012 (2012).

    This paper shows what implications size filtering may have.

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Ong, L. L. et al. Programmable self‐assembly of three‐dimensional nanostructures from 10,000 unique components. Nature 552, 72–77 (2017)

    Has example of using TEM for determining correct formation of nanostructures via electron scattering

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Schulman, R., Wright, C. & Winfree, E. Increasing Redundancy Exponentially Reduces Error Rates during Algorithmic Self‐Assembly. ACS Nano 9, 5760–5771 (2015).

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Gerling, T., Kube, M., Kick, B. & Dietz, H. Sequence‐programmable covalent bonding of designed DNA assemblies. Sci. Adv. 4, eaau1157 (2018).

    Making DNA more stable for use of designed devices

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Liu, X. et al. Complex silica composite nanomaterials templated with DNA origami. Nature 559, 593–598 (2018).

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Tikhomirov, G., Petersen, P. & Qian, L. Fractal assembly of micrometre‐scale DNA origami arrays with arbitrary patterns. Nature 552, 67–71 (2017).     :ATTACH:

    cite:tikhomirov17\_fract\_assem\_microm\_scale\_dna

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Check how many unique extensions Gregs tiles allow for.

        Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-29 Thu 10:40]</span></span>

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Review why configuration with extensions facing outwards will likely already be favored due to the lower electrostatic repulsion between the extensions-out configuration compared to one with extensions-in

<!--list-separator-->

-  DNA Origami Primer

    "Somewhat surprisingly, the first limitation is the structural design of DNA
    origami, which, to date, remains a hurdle for those new
    to DNA nanotechnology and is sometimes challenging
    even for users familiar with the technique. Although
    many design and simulation tools 8–10,15,47–50,56,59,61–63,73,267–274
    have been developed and made available to the public,
    the more versatile design tools generally require a con-
    siderable amount of user input and technical know-​how.
    In addition, the better automated tools are typically
    geared towards specific types of construct, such as 2D
    meshes with a triangulated framework 48 or wireframe
    polyhedrons 9,15,47 . Ideally, we would enjoy a suite of soft-
    ware that streamline the design and simulation process,
    serving both as a tool to easily convert simple geomet-
    rical models into DNA origami designs and as a sand-
    box for users to explore new design concepts. The field
    has made steady strides towards this goal by interfacing
    multiple design-​simulation software, developing more
    user-​friendly interfaces and allowing for post-​simulation
    touch-​up to optimize design iteratively."

    "Third, a clear picture of the DNA origami assembly
    mechanism remains elusive. Although we are equipped
    with advanced techniques to design and produce
    desired DNA origami shapes, we need to better under-
    stand how hundreds of DNA strands self-​assemble to
    form such complex structures. Besides driving a higher
    assembly yield of the target structures, the ability to
    clearly define DNA origami folding pathways will enable
    rationally designed dynamic assemblies that can toggle
    between a few metastable conformations with low
    energy barriers between them, which is a feature found
    in many protein machineries. Numerous high-​quality
    studies have shed light on this long-​standing question,
    including systematically testing DNA origami design
    variants that lead to different folding outcomes 11,69,277 ,
    measuring the global thermal transition during DNA
    origami assembly and disassembly 76 , directly observing
    assembly intermediates 98 and monitoring the incorpo-
    ration of selected staple strands 12,92 . Many of these stud-
    ies suggest a multi­stage, cooperative folding behaviour.
    Future efforts to depict such complex mechanisms will
    benefit from high-​throughput analytical methods 278
    and simulation frameworks 50,56,59,61–63,73,270,271 for DNA
    self-​assembly."

    "On the other
    hand, certain applications in physics and materials
    science, such as high-​temperature etching or 3D litho­
    graphy, can test the thermal and mechanical stability
    of DNA origami structures. In this case, the physical
    performance of DNA origami can be enhanced with a
    combination of DNA modification chemistries such as
    photo-​cross-​linking DNA nucleotides 80,286,287 , wrapping
    exposed DNA surfaces with lipid bilayers 29 , shielding
    DNA backbones with polycationic polymers 30,218,266,288–290
    and coating DNA with silica 28 . By chemically or phys-
    ically separating DNA from the environment and
    strengthening the links between DNA strands, these
    modifications help DNA origami structures survive
    low-​salt, high-​heat conditions, resist nuclease digestion,
    evade immune surveillance, prolong in vivo circulation
    and avoid surface deformation."


#### Slides {#slides}


### Questions {#questions}


#### <span class="org-todo todo HOLD">HOLD</span> How does folding path along cubic DNA block effect folding yeild? {#how-does-folding-path-along-cubic-dna-block-effect-folding-yeild}

Dependence no how degrees of freedom are constrained?
Effects by nucleotide interaction energy
Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-26 Mon 16:36]</span></span>


#### <span class="org-todo todo TODO">TODO</span> Find out what two photon 3D printing is {#find-out-what-two-photon-3d-printing-is}

Truby, R. L. & Lewis, J. A. Printing soft matter in three dimensions. Nature 540, 371–378 (2016)
"Current two-photon 3D printing affords only 200 nm

<!--list-separator-->

-  resolution and is a slow serial process"

    Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-28 Wed 23:12]</span></span>


#### <span class="org-todo todo TODO">TODO</span> Find out about current CMOS for photonics processes {#find-out-about-current-cmos-for-photonics-processes}

Sherwood‐Droz, N. & Lipson, M. Scalable 3D dense integration of photonics on bulk silicon. Opt. Express 19, 17758–17765 (2011).


#### <span class="org-todo todo TODO">TODO</span> Question: Why do metalic nanoparticles have high optical loss? {#question-why-do-metalic-nanoparticles-have-high-optical-loss}

Della Giovampaola, C. & Engheta, N. Digital metamaterials. Nat. Mater. 13, 1115–1121 (2014).


#### <span class="org-todo todo TODO">TODO</span> Find out why the \\(\lambda/8\\) limit applies/arrisses {#find-out-why-the--lambda-8--limit-applies-arrisses}

<!--list-separator-->

-  RYTOV, S. Electromagnetic properties of a finely stratified medium. Sov. Phys. JEPT 2, 466–475 (1956).

    This paper should give the answer.


### Notes {#notes}

"One key design feature enabling this success was the construction of a square DNA origami tile with multiple programmable sequences in multiple (88) precise locations along the edges of the tile 26 (Fig 3). This design creates an astronomically larger space for programming precise interactions between tiles compared to that possible with existing spherical nanoparticles decorated with only several DNA strands in poorly controlled positions. Another key to the success of our approach for increasing complexity was to hierarchically stage the assembly process, which resulted in increased yield and enabled reuse of the same set of components in multiple locations 9"


### Ideas {#ideas}


#### Reconfigurable Devices {#reconfigurable-devices}

Using tile reconfiguration (or  molecular machines) we could DNA nanocrystals for reconfiguration.
This could also maybe used to make a selectivly invert chirality on a larger DNA nanostructure when a specfic moiety (or polarization) is given.
Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-19 Mon 16:31]</span></span>


### Tasks {#tasks}


#### NEXT Read through DMMs draft and make a list with time estimates of what needs to be done for this project {#next-read-through-dmms-draft-and-make-a-list-with-time-estimates-of-what-needs-to-be-done-for-this-project}

DMM Project Draft
Also see planning book you did this in last time.


#### <span class="org-todo todo TODO">TODO</span> Read some of Co-Pi Kanté's work {#read-some-of-co-pi-kanté-s-work}

1.  Kodigala, A. et al. Lasing action from photonic bound states in continuum. Nature 541, 196–199

(2017).

1.  Bahari, B. et al. Nonreciprocal lasing in topological cavities of arbitrary geometries. Science 358,

636–640 (2017).

1.  Ndao, A. et al. Octave bandwidth photonic fishnet‐achromatic‐metalens. Nat. Commun. 11, 3205

(2020).


#### <span class="org-todo todo TODO">TODO</span> Find out whats happening with HPC? {#find-out-whats-happening-with-hpc}

Has anyone already got access to it?
Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-04-29 Thu 20:34]</span></span>


#### <span class="org-todo todo TODO">TODO</span> Find out how AFM works. {#find-out-how-afm-works-dot}


#### <span class="org-todo todo TODO">TODO</span> Find out how TEM works {#find-out-how-tem-works}


#### <span class="org-todo todo TODO">TODO</span> Find out how SEM works {#find-out-how-sem-works}


#### <span class="org-todo todo TODO">TODO</span> Review Gel Electrophoresis from Berg Biochem {#review-gel-electrophoresis-from-berg-biochem}


#### <span class="org-todo todo TODO">TODO</span> Find out why current approaches for linking nanoparticles require symmetrically inequivalent voxels {#find-out-why-current-approaches-for-linking-nanoparticles-require-symmetrically-inequivalent-voxels}

"All of the above approaches allow assembly of structures consisting of just several symmetrically inequivalent nanoparticles – far from the thousands needed for inverse design realizations"


### Aims {#aims}

DMM Project Draft
"The focus of this proposal is to develop a new nanofabrication approach that overcomes the current limitations and offers a more flexible and scalable way to construct inverse-designed 3D nanostructures"


#### 1. Synthesis of DNA-functionalized material voxels. {#1-dot-synthesis-of-dna-functionalized-material-voxels-dot}

<!--list-separator-->

-  NEXT Look at these slides (2019-09-16 Lab Genetic Imprinting Shell designs.pptx) for cube designs and categorize what approach they could be applied to (suit/imprint/both)

<!--list-separator-->

-  Considerations

    We will develop several ways to transfer the unique addressability of DNA origami from a 2D DNA-only breadboard to a 3D material voxel surface.

    <!--list-separator-->

    -  Voxel Materials

        <!--list-separator-->

        -  Losses

            Avoid metals, use something with low real part of permittivity.

        <!--list-separator-->

        -  Silica

            SiO2 at 587 nm
            refractive index n = 1.46
            relative permittivity εr = 3.9 33

        <!--list-separator-->

        -  Titania

            TiO2
            n = 2.61

        <!--list-separator-->

        -  Zinc Monoxide

            ZnO
            n = 2.00

        <!--list-separator-->

        -  Silver

            "One other voxel that can increase functionality of photonic elements 5 should be made from a material with negative real part of permittivity, such as silver (ɛ m = − 4.7 + i0.22) 32,36 . However, due to the high extinction coefficient (k = 3.9), the number of Ag voxels should be minimized in architectures that require maximum optical transparency

    <!--list-separator-->

    -  Voxel Sizes

        "For the proposed optical element design, the material voxels must be smaller than the wavelength of the light in order to suppress diffraction effects and enable the structure to behave like a homogeneous anisotropic material with an equivalent anisotropic permittivity tensor with respect to the macroscopic electromagnetic field"

        <!--list-separator-->

        -  20-100nm

            <!--list-separator-->

            -  Larger surface area advantages (still sub wavelength):

                More DNA extensions
                Require less voxels total

<!--list-separator-->

-  Deliverables

    <!--list-separator-->

    -  1.1 Voxel functionalization via origami suit approach.

        <!--list-separator-->

        -  Suit Design

            "Even though a cube is a highly symmetric shape, the cube in a suit must be endowed with
            independent addressability of each side - i.e., we need the ability to place custom DNA strands in precise positions on any of its facets."

            <!--list-separator-->

            -  Problems with;

                <!--list-separator-->

                -  Single scaffold boxes

                    Size limit (m13): 25nm^3 monolayers.
                    Using longer scaffolds: Expensive

                <!--list-separator-->

                -  DNA Tiles/Blocks

                    Lower yeild
                    More expensive

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Investigate feasibilty of Hierachial Multi-origami strategy

                Constant set of DNA sequences for any size, minimizing DNA synthesis costs

                <!--list-separator-->

                -  Testing feasibility of this approach

                    <!--list-separator-->

                    -  Assemble six DNA origami squares each with unique sequences in unique patterns

                        To probe the feasibility of a hierarchical approach, we will first assemble six DNA origami squares each with unique sequences in unique patterns (Fig. 4). Exact patterns that minimize spurious interactions will be computationally optimized.

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Insert Figure 4 from DMM draft

                        <!--list-separator-->

                        -  NEXT In small implement six squares with unique sequences (fig 4)

                            Returns: Code for creating the six squares and their their unique extensions
                            Staple sequences ready for idt (concentration and quantity) as csv file.

                            <!--list-separator-->

                            - <span class="org-todo todo TODO">TODO</span>  Find out what concentration and quantity the DNA needs to be for an idt order to make staples by the mona lisa paper.

                    <!--list-separator-->

                    -  Connect squares into a shape that can fold into a cube

                        To connect squares into a shape that can fold into a cube we will use unique edge staples similar to ones we developed for assembly of 2D arrays 9,31 . To make inter-tile connections flexible, we will insert short single-stranded domains into the edge staple sequence similar to the bridge staple design we used to connect triangles comprising the square tile (Fig. 3b, orange staples) 31 .

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Implement abstract function (generic?) to join two DNA (origami) objects

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Implement concrete function to join the sides of two DNA-tile objects

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Implement (method) function that creates squares

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Create squares with external extensions from monolayer tiles

                            Returns: File of documented code describing creation of square
                            csv file of staples
                            Experimental protocols for assembly

                            <!--list-separator-->

                            - <span class="org-todo todo TODO">TODO</span>  Design Strong and weak tile seams for cubes in fig 6a-c

                    <!--list-separator-->

                    -  Characterize

                        For ease of structural characterization with atomic force microscopy (AFM) we will first assemble a DNA origami array made of six square tiles that can unfold into a flat shape such as the one shown in Fig. 4. Suits that cannot unfold into flat structures (Fig. 5b) will be characterized with transmission electron microscopy (TEM) as we have previously demonstrated.

                        <!--list-separator-->

                        -  AFM

                            For cubes unfoldable into flat shapes.

                        <!--list-separator-->

                        -  TEM

                            For cubes NOT unfoldable into flat shapes.

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Suit Fitting

            <!--list-separator-->

            -  Extension Strands should face 'outside' the suit

                "Just as with regular clothes, without being mindful, a DNA garment can inadvertently be worn inside out. We need to avoid this so that the DNA extension strands are facing outside of the box to enable recognition of other “appropriately dressed” voxels. To make the desired inside-in topology thermodynamically favored, opposite the side with DNA extensions, we will place functional groups that have affinity to the nanoparticle surface. These functional groups should have only weak affinity to the nanoparticle surface so that intermediate complexes with incorrectly-dressed particles can rearrange into a global thermodynamic minimum which corresponds to the maximum surface contact between the suit and the nanoparticle."

                <!--list-separator-->

                -  Place functional groups with (weak) affinity to the nanoparticle surface inside the cube cavity

            <!--list-separator-->

            -  Docking the cube

                Besides ensuring that the suits wrap around particles with correct orientation, we need to bring the suits to the particles in the first place. One way to achieve the desired affinity between nanoparticle and suit is to simply take advantage of electrostatic interactions: DNA origami is negatively charged and will strive to maximize its contact with a positively charged particle surface or, by using bridging Mg 2+ ions, with a negatively charged surface 46 . Recent successes in electrostatic and hydrophobic-hydrophilic self-alignments in 2D 46–48 inspire us with confidence for our 3D particle-suit self-alignment. Even without the templating effect of the nanoparticle, the configuration with extensions facing outwards will likely already be favored due to the lower electrostatic repulsion between the extensions-out configuration compared to one with extensions-in 9"

                <!--list-separator-->

                -  Electrostatic interactions

                    <!--list-separator-->

                    -  With Nanoparticle

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Find out if titania and silca nanoparticles are negatively charged

                    <!--list-separator-->

                    -  With Extension Strands

                        extensions facing outwards will likely already be favored due to the lower electrostatic repulsion between the extensions-out configuration compared to one with extensions-in.

                        See mona lisa paper

                <!--list-separator-->

                - <span class="org-todo todo TODO">TODO</span>  Figure out experimental docking protocol

                    Buffer/Solution
                    Concentration Nanoparticle
                    Concentration Suits (in what stage)
                    Steps and Conditions (Temp, duration etc)

                    See [Yeild](#yeild) for the strong weak seams needed for complexation steps

            <!--list-separator-->

            -  Yeild

                <!--list-separator-->

                -  Partially form suits before nanoparticle complexation

                    <!--list-separator-->

                    -  Strong and weak seams

                        The cube may be in its empty and completely closed form at room temperature, but opens up to allow a particle in at higher temperature as long as that temperature is below the melting point of strong seams and the origami tile itself (~55°C) 31 .

                        <!--list-separator-->

                        -  Inter tile seams

                            The inter-tile seams in these structures can
                            be very strong, e.g., with melting points
                            above 50°C.

                        <!--list-separator-->

                        -  Remaining Seams

                            The remaining seams to complete the cube need to be weak enough to stay open at the assembly temperature.

                        <!--list-separator-->

                        -  Designs

                            "A cube made of 6 square tiles has 12 seams. The maximum number of strong
                            seams that allow an open cube is 9 (Fig. 6c). It is likely that the higher the number of strong seams, the higher the encapsulation yield will be. We will first study nanoparticle encapsulations for the structures shown in Fig. 6a-c"

                            <!--list-separator-->

                            -  Likely that the higher the number of strong seams, the higher the encapsulation yield will be

                            <!--list-separator-->

                            - <span class="org-todo todo TODO">TODO</span>  Insert fig 6

                        <!--list-separator-->

                        - <span class="org-todo todo TODO">TODO</span>  Determine how long (nts) strong and weak seams need to be

                            See both of Gregs paperst that have tiles in them.

        <!--list-separator-->

        -  Size and Shape Tolerance

            <!--list-separator-->

            -  Size exculsion by suit fitting

                <!--list-separator-->

                -  Purification: Gel Electrophoresis

                    Running gel electrophoresis on a mixture of naked and dressed particles, naked particles can be removed due to their lack of a negatively-charged DNA suit. In this way, the dressed particles can be efficiently purified for further assembly

                <!--list-separator-->

                - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocol on how to purify nanoparticles using DNA suits

                    Suit Designs
                    Which nanoparticles
                    Suit assembly
                    Characterization
                    Theoretical predictions

    <!--list-separator-->

    -  1.2 Voxel functionalization via DNA imprinting

        In this strategy ,alternative to [Aim 1.1](#1-dot-1-voxel-functionalization-via-origami-suit-approach-dot), a DNA origami suit is put on the particle “inside out” (with extensions facing inside, not outside as in Aim 1.1) and then removed
        leaving behind the extensions (“imprints”) in precise locations

        <!--list-separator-->

        -  Advantages

            <!--list-separator-->

            -  Smaller Gap

                More intimate contact between the particles would allow better electronic coupling beneficial for certain applications.

                <!--list-separator-->

                - <span class="org-todo todo TODO">TODO</span>  Cheben, P., Halir, R., Schmid, J. H., Atwater, H. A. & Smith, D. R. Subwavelength integrated photonics. Nature 560, 565–572 (2018)

                    For example smaller gaps between two plasmonic particles result in higher electric field enhancement in the gap

            <!--list-separator-->

            -  Broader range of assembly conditions

                Higher Temp

            <!--list-separator-->

            -  Challenges

                <!--list-separator-->

                -  Extensions-in configuration

                    To favor extensions-in assembly of the suit, we cannot rely on electrostatic
                    interactions that we proposed for extensions-out suit fitting, since extensions-out will be naturally favored due to electrostatic repulsion between the extensions 50 and the “entropic brush” effect 46,48

                    <!--list-separator-->

                    - <span class="org-todo todo TODO">TODO</span>  What is entropic brush effect

                    <!--list-separator-->

                    -  Proposed Solution

                        Functionalize the extensions to have a specific sequence-defined affinity to the surface of nanoparticles. A new challenge here is that this affinity needs to be made weak while the suit is being put on the particle to reach the best alignment and then strong to retain the DNA strands when the suit is being removed. There are many
                        ways to achieve this weak-to-strong transformation. We next present one example strategy (Fig. 7b).

                        <!--list-separator-->

                        -  Step 1: Coupling extensions to nanoparticle strategies

                            <!--list-separator-->

                            -  Proposed Method: Covalent cojugation via amine coupling

                                DNA strands hybridize to the particle and become conjugated via amide coupling. The suit is then removed by melting or strand displacement.

                                <!--list-separator-->

                                -  1.1 Weak bond creation (fitting)

                                    To create weak interactions, we will use nanoparticles coated with short DNA strands, e.g., T 3 corresponding to the complementary A 3 terminus on the DNA origami extensions. Suit alignment will be driven by maximization of the number of T 3 :A 3 contacts.

                                <!--list-separator-->

                                -  1.2 Strong bond creation (imprinting)

                                    Alignment will bring the terminal amine groups near the particle surface where they will be covalently linked to the carboxyl groups via standard aqueous amide coupling protocols

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Nakajima, N. & Ikada, Y. Mechanism of Amide Formation by Carbodiimide for Bioconjugation in Aqueous Media. Bioconjug. Chem. 6, 123–130 (1995).

                                        This paper give the coupling chemistry

                                    <!--list-separator-->

                                    -  Getting T3-coated nanoparticles

                                        T3 -coated particles can be easily obtained by conjugating a carboxyl-coated nanoparticle surface with amine-terminated DNA strands (T 3 -NH 2 , amine is a commercially available terminus modification available from Integrated DNA Technologies)

                                        <!--list-separator-->

                                        - <span class="org-todo todo TODO">TODO</span>  Find sources and cost them

                                <!--list-separator-->

                                - <span class="org-todo todo TODO">TODO</span>  Insert Image

                                <!--list-separator-->

                                - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for covalent conjugation

                        <!--list-separator-->

                        -  Step 2: Removal of Suit Strategies

                            <!--list-separator-->

                            -  Melting

                                Above origami melting point

                            <!--list-separator-->

                            -  Strand displacement

                                <!--list-separator-->

                                - <span class="org-todo todo TODO">TODO</span>  Think about advantages of this vs melting or other methods

                            <!--list-separator-->

                            - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for suit removal

                        <!--list-separator-->

                        -  Step 3: Purification Strategies

                            <!--list-separator-->

                            -  Gel Electrophoresis

                                "Now the suit can be melted off by heating above the origami melting point (~55°C), followed by gel electrophoresis purification, leaving behind the covalently attached “imprints” (Fig. 7b2). Thus the imprints will consist of three domains: A 3 glue, a sequence that previously attached the imprint to the DNA origami scaffold, and a linker with arbitrary sequence between them. The last two domains will be used to program voxel self-assembly."

                            <!--list-separator-->

                            - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for purification

                        <!--list-separator-->

                        -  Step 4: Structral Characterization

                            <!--list-separator-->

                            -  Flat Suits

                                "We will characterize flat DNA suits with atomic force microscopy (AFM) to validate tile connectivity and DNA extensions on each tile 9,31."

                                <!--list-separator-->

                                -  AFM

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for characterization via AFM

                            <!--list-separator-->

                            -  Non-flat suits

                                "We will characterize 3D architectures such as non-flatsuits, nanoparticles, and nanoparticle inside suits by transmission electron microscopy (TEM) and scanning
                                electron microscopy (SEM)."

                                <!--list-separator-->

                                -  TEM

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for characterization via TEM

                                <!--list-separator-->

                                -  SEM

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Come up with experimental protocols for characterization via AFM

                            <!--list-separator-->

                            -  Efficiency of strand transfer (imprinting approach)

                                To characterize how well DNA strands transfer from suit to nanoparticle, we will
                                use three approaches.

                                <!--list-separator-->

                                -  3D DNA PAINT (Point Accumulation In Nanoscale Topography)

                                    First, we will perform 3D DNA PAINT (Point Accumulation In Nanoscale Topography) that will directly allow imaging of attachments via transient binding to fluorescently-labeled complementary DNA strands

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Jungmann, R. et al. Multiplexed 3D cellular super‐resolution imaging with DNA‐PAINT and Exchange‐PAINT. Nat. Methods 11, 313–318 (2014).

                                <!--list-separator-->

                                -  Hybridize voxels with gold nanoparticles bearing complementary DNA strands and image the product with TEM and SEM

                                    If DNA strand has transferred, we should see an easily visible Au nanoparticle attached in that location.

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  Readup more on method

                                <!--list-separator-->

                                -  Self-assemble two or more DNA-functionalized voxels in programmed shapes.

                                    We will test assembly of simple shapes such as dimers, linear/non-linear trimers, and 2×2, 2×2×2 arrays with SEM and TEM.

                                    <!--list-separator-->

                                    - <span class="org-todo todo TODO">TODO</span>  In small create some (tetris?) blocks from voxels

                                        Code
                                        Staple Seq
                                        Experimental protocols


#### 2. Self-assembly of voxels into photonic elements. {#2-dot-self-assembly-of-voxels-into-photonic-elements-dot}

"We will develop several strategies to self-assemble the DNA-functionalized voxels into desired nanophotonic architectures."

<!--list-separator-->

-  2.1 Design of inter-voxel bonds

    We will explore three ways of constructing DNA bonds (Fig. 8).

    <!--list-separator-->

    -  Method 1: Direct Hybridization (both 5' ends attached to nanoparticle)

        we will form bonds by directly hybridizing DNA strands on one voxel to those on another, with strands attached to both nanoparticles via the 5’ end (Fig. 8a).

    <!--list-separator-->

    -  Method 2: Direct Hybridization (one 3' and one 5' ends attached to nanoparticle)

        we will also hybridize the particles directly, but now one of the particles will have strands attached via the 3’ end (Fig. 8b).

    <!--list-separator-->

    -  Method 3: Bridging Strands

        We will use bridge strands that will hybridize to strands on both nanoparticles (Fig. 8c).

    <!--list-separator-->

    -  Comparison of methods

        The bonds produced by the first method are most commonly used to connect nanoparticles 18,19,52 . However, for our purposes of building optical nanoscale systems, it may be desirable to reduce the gap between voxels, which Method 2 allows, due to the antiparallel requirement for DNA duplex formation (Fig. 8b). Furthermore, bonds produced by Method 2 are more tolerant to variation in inter-particle spacing compared to Methods 1 and 3, as even nanoparticles with some surface roughness will be able to bond, albeit with less base pairs, making the design experimentally more robust. Finally, Method 3 bonds require bridge strands to be added for the voxels to bind, reducing the set of unique voxels needed to assemble complex architectures. By changing bridge sequences, the same voxel can be programmed to bind to many different voxels, not just the one that with matching sequences as in Methods 1 and 2.

    <!--list-separator-->

    -  Experimental Questions

        "We will explore the effect of length and sequence of single-stranded linkers on voxel self-assembly, taking advantage of the wealth of experimental data and rules elucidated for more symmetrically DNA-functionalized nanoparticles 18,19,52,53 . Importantly, the unique spatial addressability of DNA origami creates the astronomical programming space that is needed for constructing large assemblies from thousands of unique voxels. For example, with just two different sequences we can create 2^136 ~ 8.7×10 40 different patterns on a single side of a 92 nm cube, since the corresponding DNA origami square (Fig. 2) has 136 unique extendable locations 9 . To avoid kinetic traps, we will use 5-nucleotide or shorter DNA strands because longer sequences would be too strong to allow thermodynamically controlled assembly 31 . Although 136 is the maximum number for a 92 nmsquare, we anticipate using ~ 20 different strands will be sufficient to achieve necessary specificity for typical applications.

        Unlike lithographic nanofabrication that generates monolithic static structures, assembly with DNA could enable reconfiguration capability in response to external stimuli via strand displacement 54,55 . This feature can be used to program switching between multiple functions by reconfiguring the multivoxel element."

        <!--list-separator-->

        -  Effect of length and sequence of single-stranded linkers on voxel self-assembly

        <!--list-separator-->

        -  Number of required extensions

<!--list-separator-->

-  2.2 Algorithms for voxel self-assemblyH

    Ideally we would like to identify algorithms that enable voxel self-assembly in the smallest number of steps, at lowest cost, and with maximum yield. We next consider strategies for optimizing each of these parameters.

    <!--list-separator-->

    -  Approaches

        <!--list-separator-->

        -  One-step assembly

            "The smallest number of steps would be achieved by one-step assembly of all 4,096 (16 3 )
            voxels into the final architecture shown in Fig. 1. In principle, DNA nanotechnology now allows self-assembly
            of structures from 10,000 unique components in a single step 43 . However, this would require assembly of 4,096
            voxels each functionalized with ~ 6 unique “glues”. “Glue” here refers to a specific facet-to-facet interaction.
            More specifically, every internal cubic voxel needs 6 glues, while surface voxels need only 5 (facet), 4 (edge),
            or 3 (corners) glues (Fig. 1). As discussed in the previous section, this large number of glues is achievable
            given the astronomical design space enabled by programming DNA sequences and their positions. However,
            substantial time would be needed to mix DNA strands for each of the 4,096 voxel suits. Even using the currently
            fastest LabCyte Echo ultrasonic robotic liquid handler, it takes ~ 30 minutes to mix 64 unique origami squares
            (~ 20,000 strands) 9 , which is approximately equivalent to 10 suits for cubes (each cube is made of six squares). Thus creating 4,096 voxel suits would take ~200 hours."

        <!--list-separator-->

        -  Assembly with minimum number of glues.

            "The number of required glues can be dramatically reduced by taking a multistep assembly strategy. In its extreme – 4,096-step assembly – each voxel is added to the mix stepwise. This would require a number only on the order of the number of symmetrically inequivalent voxels which is 4 (internal, facet, edge, corner). This process would be akin to 3D printing where a single voxel is added at a time (but unlike 3D printing, we can assemble trillions of copies in parallel and we can control each voxel’s material precisely). A concern with this multistep process is that, similar to multistep non-biological synthesis of DNA on robotic synthesizers, an excess of voxels would be needed at each stage. Depending on how large and recoverable that excess turns out to be, this scheme may be costlier than others"

        <!--list-separator-->

        -  Fractal assembly

            "A hierarchical multi-step approach can provide an optimal trade-off for minimizing both number of steps and number of glues. One such recursive scheme, “fractal assembly,” uses the same assembly rule at every stage and multiple “pots” leading to a 16×16×16 architecture in just four steps (Fig. 9) with significantly less glues than required for a single step assembly. This protocol is the 3D implementation of our 2D fractal assembly that enabled the largest ever uniquely-addressable DNA canvas 9 (Fig. 3). For an N×N×N array, the scheme requires \\(\log\_2 N\\) steps and \\(O(N)\\) glues (i.e., the number of glues scales linearly with  N). In the first step, 4,096 voxels self-assemble into groups of 8 in 512 different test tubes to yield 2×2×2 cubes  (Fig. 9a→b). These larger cubes are combined again in groups of 8 in 64 different test tubes to self-assemble into 4×4×4 cubes (Fig. 9b→c), and so on until final architecture (e) is reached. Even though the location of each voxel is uniquely encoded, this scheme allows reuse of the same glues in different test tubes, thus greatly reducing the total number of unique glues and voxels."

<!--list-separator-->

-  2.3 Structural characterization of assemblies.

    "It is important to validate that desired structures assemble correctly and with minimal defects (e.g., without missing or incorrectly incorporated voxels). As we discussed in Aim 1, we will use voxels made of TiO 2 and SiO 2 to construct our optical metamaterials. Since Ti (atomic mass 47.8) is heavier than Si (atomic mass 28.1), TiO 2 voxels should scatter electrons more strongly than SiO 2 and thus appear darker in transmitted projections in TEM images. By looking at all three orthogonal projections of a cube it should in principle be possible to validate if the number of each type of voxel in each column is consistent with the design 43 . In addition, SEM allows us to tell apart different surface voxels based on intensity of backscattered electrons, i.e., TiO 2 voxels should appear brighter than SiO 2 voxels. While SEM and TEM will give nanoscale details, gel electrophoresis should give us relative populations of each structure. In addition, gel electrophoresis will allow us to purify intermediates for the next step if the yield is low. We will first characterize simple 2×2×2 cubes by gel, SEM, and TEM and then more complex structures. Similar to our previous work on fractal assembly of 2D arrays 9 , we will optimize experimental conditions to obtain the best yield of 4×4×4 arrays and then use what we have learned to build bigger structures.


#### 3. Design and characterization of optical functions for multivoxel architectures. {#3-dot-design-and-characterization-of-optical-functions-for-multivoxel-architectures-dot}

"We will construct architectures with several specific functions in optical and IR windows and characterize them."

<!--list-separator-->

-  3.1 Color image sensors with tripled quantum efficiency

    "In conventional image sensors on digitalcameras, ~2/3 of light is lost by absorptive color filters before being detected. We propose to self-assemble a 3D dielectric scatterer, designed to be placed on top of an RGBG quartet of pixels in an image sensor, that sorts and focuses light based wavelength to achieve efficiency significantly surpassing 2D absorptive and diffractive filters (Fig. 10a). Two architectures for building such a scatterer were identified by Camayd-Muñoz et al using inverse design 4 ; starting inverse optimization with 20 nm voxels, the authors found that no features smaller than 60 nm were present in the final optimized architecture 4 . Despite these theoretical calculations, the authors could only construct a much larger-scale device operating at microwave frequencies due to fabrication limitations. Our approach is perfectly positioned to provide a scalable and precise method for fabricating image sensors in the visible range surpassing the limitations of 2-photon 3D printing and multilayer lithography. Our origami methods (Fig. 4, 7) are well-suited to self-assemble 60-nm voxels with appropriate materials (silica and titania). Preliminary inverse optimizations using 60 nm cubic voxels and 5 nm intervoxel gaps filled with silica carried out at the high performance computing center at Caltech with the existing software packages achieve close to reported 4,14 optical element performance.

    For testing the 3D dielectric scatter, we will illuminate an element on a transparent substrate with a broadband
    light source. We will build a microscope using a high numerical aperture (NA) and high magnification objective
    lens and appropriate sample stages to image the light from the scatterer. Then, at the magnified imaging plane,
    we can put a pinhole at one of the four quadrants shown in Fig. 10a, and measure the signal through the
    pinhole with a spectrometer. Alternatively, instead of the broadband source and spectrometer, we can use a
    laser with tunable wavelength and a photo-detector. Only one of six possible orientations of the cube will give
    the desired performance; a way to achieve desired orientation and location of the cubes is described in Aim 4.
    These experiments will be done in the lab of Co-PI Kanté, who has the necessary expertise and equipment."

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Do simulations for suit and imprint designs for Bayer Filter

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Follow up with Greg on using his Bayer filter sims as a starting point

            Follow up on this mail "Re: question about inverse design"

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Insert Figure 10

<!--list-separator-->

-  3.2 Finite photonic crystals with arbitrary symmetry

    "For many photonics applications such as on-chip photon sources, finite photonic crystals 58 (PC) are needed with precise size, location, and orientation on a chip. Current PC self-assembly approaches typically use one or two spherical components or block copolymers that only yield unbounded arrays with limited symmetries 21 . We will develop self-assembly of finite photonic crystals with arbitrary symmetries enabling a host of new transformative applications.

    Since our assembly scheme allows arbitrary placement within a cubic lattice comprised of low and high
    refractive index voxels, we can in principle access any cubic symmetries, e.g., the ones shown in Fig. 10d-i.
    These include topologies not accessible with soft lithography techniques such as woodpile-based soft
    lithography 59 (Fig. 10g). These also includes a diamond lattice (F4̅ 3m, No. 216, Fig. 10h) that exhibit an almost
    complete photonic bandgap in the visible regime 60 . We can potentially shift layers or columns of voxels relative to each other to access all hexagonal symmetry groups (Fig. 10i). The dynamic nature of the DNA inter-voxel
    bonds can be used to enable PCs with reconfigurable structures (e.g., 10f↔810i) and tunable bandgap, a
    dream of many physicists and engineers. Moreover, our approach can enable engineering photonic cavities
    with designer symmetry 61 , arbitrary location, and even controlled emitter placement within the cavity 47 .

    To characterization the PCs we will use a broadband light source, a rotation mount, focusing lens, and a
    spectrometer, and additional equipment depending on the attribute being measured. We will collaborate with
    the lab of the photonic crystals pioneer Eli Yablonovitch at UC Berkeley to design and characterize PCs (see
    letter of support)."

    <!--list-separator-->

    -  Reference Material

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Freymann, G. von, Kitaev, V., Lotsch, B. V. & Ozin, G. A. Bottom‐up assembly of photonic crystals. Chem. Soc. Rev. 42, 2528–2554 (2013).

        <!--list-separator-->

        -  NEXT Joannopoulos, J. D., Villeneuve, P. R. & Fan, S. Photonic crystals. Solid State Commun. 102, 165–173 (1997).

            Ref giving finite photonic crystal (aim 3.2)

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Lee, J.‐H. et al. Semicrystalline woodpile photonic crystals without complicated alignment via soft lithography. Appl. Phys. Lett. 96, 193303 (2010)

            Woodpile Lithography reference

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Maldovan, M. & Thomas, E. L. Diamond‐structured photonic crystals. Nat. Mater. 3, 593–600 (2004).

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Zheng, X., Taverne, M. P. C., Ho, Y.‐L. D. & Rarity, J. G. Cavity Design in Woodpile Based 3D Photonic Crystals. Appl. Sci. 8, 1087 (2018).

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  [Gopinath, A., Miyazono, E., Faraon, A. & Rothemund, P. W. K. Engineering and mapping nanocavity emission via precision placement of DNA origami. Nature 535, 401–405 (2016)](#gopinath-a-dot-miyazono-e-dot-faraon-a-dot-and-rothemund-p-dot-w-dot-k-dot-engineering-and-mapping-nanocavity-emission-via-precision-placement-of-dna-origami-dot-nature-535-401-405--2016)

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  In small implement function to create structures with arbitrary symmetry for a given approach (suit/imprint)

        <!--list-separator-->

        -  NEXT Look at all structures in figure 10 and categorize their defining features (lattice vectors)

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Develop experimental protocols

        Synthesis
        Purification
        Stabilization
        Characterization

        :Effort:   2:00

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Collect all info on how we would test this and review it

            Emails, DMM draft

<!--list-separator-->

-  3.3 Finite plasmonic metacrystals

    It has been theoretically predicted that a lattice composed of metallic particles such as silver and gold can be designed to exhibit an arbitrary combination of electric permittivity (ε r ) and magnetic permeability (µ r ). Simply by tuning the size and spacing between cubic particles, optical effective media with refractive indices and impedances can be achieved that are unattainable for any existing natural materials 62 or existing general purpose experimental fabrication approach. We will develop self-assembly of plasmonic crystals with arbitrary compositions enabling a host of new transformative applications, such as materials with high refractive index (𝑛 √ ε r µ r ) that are of particular interest for imaging and lithography, where the resolution scales inversely with the refractive index.

    Previously a metacrystal made of plasmonic nanoparticles and dielectric media filling the gaps between them
    was shown to exhibit a dielectric Mie resonance (Fig. 10j, k) 57,62,63 . This resonance in effective permeability
    strongly depends on the size of the crystal 57 which could be imprecisely controlled by tuning crystal growth
    conditions. With our approach we will assemble finite crystals with exactly specified dimensions, which
    would enable designer magnetic resonances in the visible range of the electromagnetic spectrum and ultimate
    control over the optical properties of plasmonic metamaterials. As discussed in Aim 1, we can easily apply our
    DNA imprinting approach not only to dielectric voxels such as silica but also to plasmonic voxels such as silver.

    We will measure the properties of our metacrystals in bulk using small angle X-ray scattering 18 , and
    characterize single metacrystals with TEM and SEM to validate size, quality, and nanoparticle spacing, electron
    energy loss spectroscopy to analyze electric field distribution in the gaps between metal particles, and use
    focused synchrotron beams to measure magnetic resonances 57 .

    <!--list-separator-->

    -  Reference Material

        <!--list-separator-->

        -  NEXT Chung, K., Kim, R., Chang, T. & Shin, J. Optical effective media with independent control of permittivity and permeability based on conductive particles. Appl. Phys. Lett. 109, 021114 (2016).

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Meinzer, N., Barnes, W. L. & Hooper, I. R. Plasmonic meta‐atoms and metasurfaces. Nat. Photonics 8, 889–898 (2014).

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Kim, S. et al. Mie‐Resonant Three‐Dimensional Metacrystals. Nano Lett. 20, 8096–8101 (2020).

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Find out what Mie-Resonance is

    <!--list-separator-->

    -  Experimental Protocols

        <!--list-separator-->

        -  Design

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Find out how these crystals are designed

                Read the papers
                Document design process

        <!--list-separator-->

        -  Characterization

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Find out what Small angle xray scattering is

                What it measures
                How its preformed
                What is its resolution
                What equipment is necessary
                What data processing is needed
                What volume of sample is needed

                <!--list-separator-->

                - <span class="org-todo todo TODO">TODO</span>  Park, S. Y. et al. DNA‐programmable nanoparticle crystallization. Nature 451, 553–556 (2008).

                    This paper shows how we use SAXS for DNA-nanoparticle characterization.

                    :Effort:   2:00

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Find out what electron energy loss spectroscopy is

            <!--list-separator-->

            - <span class="org-todo todo TODO">TODO</span>  Find out how focused synchrotron beams are used to measure magnetic resonances

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Find out why in imaging and lithography the resolution scales inversely with the refractive index

        Entered on <span class="timestamp-wrapper"><span class="timestamp">[2021-05-01 Sat 16:39]</span></span>


#### 4. Precision placement of optical elements in existing devices. {#4-dot-precision-placement-of-optical-elements-in-existing-devices-dot}

"In this final exploratory Aim of our proposal, we consider how the new self-assembly approach developed in Aims 1-3 can be translated into industrially-relevant technologies. Specifically, we discuss the challenge of how to integrate our optical components into practically-useful devices. We will commence with this Aim only after achieving the fundamental technology development goals of Aims 1-3.

Our self-assembled optical components (e.g., color sorter, finite photonic crystal, Mie-resonant metacrystal) would be most useful when placed in precise location in precise orientation onto existing device platforms (e.g., image sensor array, light-emitting diode arrays, imaging lens). In general, the problem of integrating bottom-up components with conventional top-down fabricated wafers is a huge challenge limiting practical implementations of most self-assembled materials and devices. In particular, current industrial techniques for precision placement such as robotic assembly are infeasible due to the small size of our components. For example, robotic pick-and-place, although accurate and reliable at larger scales, is a serial process that downscales unfavorably due to stiction problems, fragility, and the sheer number of components in a device such as an image sensor. Fortunately, a recent report provides a glimpse of hope. Gopinath et al. demonstrated that it is possible to align certain DNA origami shapes onto E-beam-patterned substrates with matching hydrophilic patterns 48 . This self-alignment is nanometer-precise, scalable (i.e., trillions of identical shapes are aligned in parallel), and yields one unique orientation. The latter is a must for applications such as color image sensors, since the RGBG grid is asymmetric (Fig. 10a).

In a proof-of-concept to demonstrate how we can integrate our photonic elements onto existing devices, we
propose to adapt the approach of approach Gopinath et al. to place color splitters produced by self-assembly (Fig. 10a-c) on top of a conventionally-fabricated image array. First, we will make a round patch with asymmetrically-placed circle by photolithographic patterning of a silicon substrate coated with a hydrophobic layer such as trimethysilyl 46,48 (Fig. 9a). We will then make the patch hydrophilic by etching with oxygen plasma (Fig. 9b) apnd stripping the resist (Fig. 9c). Then we will add a solution of a micron-sized DNA arrays that we have recently described 9 with a matching extension pattern on its bottom so it can self-align with the hydrophilic pattern (Fig. 9d). The top of the origami can have an arbitrary strand pattern (shown as Mona Lisa in Fig. 9d). We will then use the top as a “smart surface” to dock our DNA strand-decorated color splitter (Fig. 9e).

We will also test if it is possible to align color splitters that have matching DNA extension patterns on the bottom directly without first placing origami. However, the DNA origami adapter approach is more similar to the reported protocol 51 and will likely be more general, enabling placement of multiple self-assembled components onto the same micron-sized area. One big difference between our approach and that of Gopinath et al. is that we are using a 64 times larger DNA array. This should enable highly scalable and accessible optical wafer patterning rather than slow, serial E-beam lithography. We note that image sensors are a particularly suitable application for self-assembly, because thousands of identical components need to be organized into a periodic lattice (Fig. 10a). More broadly, embedding DNA recognition into surfaces patterned with optical lithography will enable precision incorporation of arbitrary nanoscale materials into existing devices opening a host of previously impossible functions."

<!--list-separator-->

-  Reference Material

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Electrostatic and hydrophobic-hydrophobic aligmnment of DNA

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Gopinath, A. & Rothemund, P. W. K. Optimized Assembly and Covalent Coupling of Single‐Molecule DNA Origami Nanoarrays. ACS Nano 8, 12030–12040 (2014).     :ATTACH:

        <!--list-separator-->

        - <span class="org-todo todo TODO">TODO</span>  Gopinath, A., Miyazono, E., Faraon, A. & Rothemund, P. W. K. Engineering and mapping nanocavity emission via precision placement of DNA origami. Nature 535, 401–405 (2016)     :ATTACH:

        <!--list-separator-->

        -  NEXT Gopinath, A. et al. Absolute and arbitrary orientation of single molecule shapes. (2018)

<!--list-separator-->

-  Design

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Create DNA models in small for the tile array and

<!--list-separator-->

-  Experimental Protocols

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Find out how to make a round patch with asymmetrically-placed circle by photolithographic patterning of a silicon substrate coated with a hydrophobic layer such as trimethysilyl (Fig. 9a)

    <!--list-separator-->

    - <span class="org-todo todo TODO">TODO</span>  Find out how to make the patch hydrophilic by etching with oxygen plasma (Fig. 9b) and stripping the resist (Fig. 9c).

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  Insert figure 9 from DMM draft

<!--list-separator-->

- <span class="org-todo todo TODO">TODO</span>  What is a "conventionally-fabricated image array"


#### Synthesis of material voxels inside DNA suits {#synthesis-of-material-voxels-inside-dna-suits}

"One potential limitation of the approaches described in Aim 1 is
reliance on available cubic nanoparticles. The materials that are needed for the described applications (TiO 2 , SiO 2 , Ag, Au) are already available in various cubic particle sizes and crystalline forms (e.g., rutile, anatase, brookite for TiO 2 ). However, if this self-assembly approach is successful, to build devices for applications beyond nanophotonics, a larger variety of materials in different sizes and shapes will be needed. Furthermore, the capability to use other materials will greatly increase the flexibility of our approach for nanophotonics, e.g., enabling smaller elements that better approximate desired optical functions.

To address this potential need, we propose to grow material voxels inside DNA suits (Fig. 12). We will functionalize the interior surface of the suit with precisely-placed patterns of molecular seeds (green “S” in Fig. 12a). These seeds will serve as nucleation sites for the assembly of low molecular weight precursors inside the shell. The seeds can be varied to nucleate voxels with a wide range of optical parameters, e.g., N-trimethoxysilyl group to nucleate silica 64 , phosphorothioate DNA that we reported 8 to nucleate metallic or semiconducting voxels, and initiators for atom transfer radical polymerization (ATRP) 65 to nucleate transparent conducting or semiconducting polymer particles. Even applications using SiO 2 and TiO 2 voxels can benefit from this approach since the voxels can potentially be made much more uniform compared to commercially-available nanoparticles.

Designing a DNA suit in a custom shape is a solved challenge for almost arbitrary-shaped particles under 50 nm 27–30,43 , but few of the proposed structures would be mechanically stable enough to withstand the large mechanical outward force generated by growing inorganic particles 66 . To make mechanically stable cubic suits we will use double layer square origami with DNA strands orthogonal between the two layers 67 (similar to plywood), which should be mechanically stronger than multilayer suits with parallel helices 66 (Fig. 12d)."


#### Cost {#cost}

"Many proponents of DNA nanotechnology like to point out that the cost of DNA is decreasing faster than
Moore’s law. It is true the cost is decreasing per unique base (currently at ~$10 -10 per base) on a nanomole scale, which is important for traditional molecular biology and emerging DNA data storage applications 68 . For mass manufacturing of optical components that we propose, it is more important to estimate the cost per gram of the same sequence, as all that is needed is many copies of the same sequence. Luckily, this cost is also being dramatically scaled down. By genetically encoding both scaffold and staple strands, it now requires just seven liters of culture to yield a gram of DNA origami 69 ; at 800-liter scale, total costs, including equipment, labor, and energy are $200 per gram. A single gram of DNA origami can cover 1000 m 2 , more than three tennis courts in area or 6,250 of 450 mm (largest size) silicon wafers. Each photonic element has 4,096 (16×16×16) voxels. Each cubic 60 nm voxel has surface area equal to 21,600 (60×60×6) nm 2 . Thus each photonic element will need a maximum of ~8.8×10 7 (4,096×21,600) nm 2 of DNA origami. Professional camera sensors now have 50×10 6 pixels (50 megapixels) and will thus need 4.4×10 15 nm 2 of origami. This will cost 4.4×10 15 nm 2 ×$200/(1000 m 2 ×10 18 nm 2 /m 2 ) = $8.8×10 -4 or ~0.1 cent per entire 50 Megapixel sensor in DNA costs. It is hard at this stage to estimate the labor costs for mass production of the photonic elements; we only note that the self-assembly process will not require expensive state-of-art nanofabrication facilities but can be done on a standard lab benchtop."


#### Yeild {#yeild}

The percent of devices on the wafer found to perform properly is referred to as the yield in semiconductor device fabrication. High yields are needed to fabricate economically viable products. The yield of a typical wafer patterning process varies from 30 to 90%. For example, TSMC announced an average yield of ~80% 70 , with a peak yield per wafer of >90% for their 5 nm test chips with a die size of 17.92 mm 2 and 32.0% when die size was increased to 100 mm 2 . The yields in DNA self-assembly vary from ~98% for a single-layer origami 9 to ~1% for complex multi-layer 3D architectures assembled from over 10,000 components 43 . Some DNA nanotechnology skeptics quote low yields as a fatal flaw that will prevent integration of DNA techniques with semiconductor manufacturing. It is important to note, however, that self-assembly can be scaled much more easily than lithography. For example, it costs almost the same time, money, and energy to self-assemble 100 trillion copies of the target ensemble (~0.01 nanomoles) as 100 million (10 -8 nanomoles). A single nanomole of DNA-enabled devices will likely be enough to satisfy all of humanity’s demand for that particular optical component (e.g., supplying filters for 5 million iPhones). In addition, we can use powerful, massively parallel tools for quality control that were developed for the pharmaceutical industry – a giant sector with revenue and quality standards rivalling the semiconductor industry. These quality control tools include purification techniques such as size exclusion chromatography, rate zonal centrifugation, and gel electrophoresis. Unlike serial and lengthy die testing in the semiconductor industry, these purification techniques allow astronomical numbers of molecules or molecular devices to be purified in parallel. We are planning to use gel electrophoresis and glycerol-gradient centrifugation 71 to purify our assemblies at low and large scales respectively. If necessary, we will perform purification on early assembly stages (not just the last one) to improve the integrity and yield of the final assemblies.


#### Defect tolerance {#defect-tolerance}

Our optical elements should have a much higher tolerance to the presence of defects such as missing DNA strands, voxels, or even an entire color splitter compared to semiconductor devices. In an integrated circuit, one bit flipped from 0 to 1 by cosmic radiation halted a giant Google server 72 . In contrast, we can design our assembly scheme to be robust to missing DNA strands 73 . A few missing voxels will likely have imperceptible effect on image quality (to be tested computationally and experimentally). Image sensors with an entire color splitter missing can be eliminated by existing manufacturing quality controls. Moreover, the dynamic nature of DNA bonds may enable device self-repair and upgrade one day, e.g., using a process such as tile displacement that we have developed 44


#### Stability {#stability}

DNA is an organic material incompatible with many processes used in semiconductor manufacturing and packaging. Luckily, a host of techniques to improve the stability of the architectures assembled with DNA have been reported 64,74,75 . To improve the stability of an individual origami or multi-origami box, we can use T- T welding 74 . To improve the stability of multivoxel photonic elements, we can encapsulate them in silica 75 . Beyond these steps, standard device packaging technologies can be applied. DNA can be removed from the final device by plasma treatment.
